runtime/ms_rest/lib/ms_rest/jsonable.rb

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+module MsRest
+  # Mixin to provide simple serialization / deserialization in AutoRest generated model classes
+  module JSONable
+    include MsRest::Serialization
+
+    def to_json
```

Comment:
Good point. I'll update the pull request with documentation. Thank you for catching this.